youtube_dl/extractor/bitchute.py

```diff
@@ -59,8 +61,14 @@ def _real_extract(self, url):
             for format_url in orderedSet(format_urls)]
 
         if not formats:
-            formats = self._parse_html5_media_entries(
-                url, webpage, video_id)[0]['formats']
+            entries = self._parse_html5_media_entries(
+                url, webpage, video_id)
+            if not entries:
+                error = self._html_search_regex(r'<h1 class="page-title">([^<]+)</h1>', webpage, 'error', default='Cannot find video')
+                if error == 'Video Unavailable':
+                    raise GeoRestrictedError(error)
```

Comment:
This does not necessarily mean geo restriction, they may have other reasons for video unavailability. It's better checking the message inside `p` that explicitly states about blocking content in `your country`.

Reply:
Like that?